src/redux/contacts.ts

```diff
@@ -51,11 +117,11 @@ export const removeContact = address => (dispatch, getState) => {
 };
 
 // -- Reducer ----------------------------------------- //
-const INITIAL_STATE = {
+const INITIAL_STATE: ContactsState = {
   contacts: {},
 };
 
-export default (state = INITIAL_STATE, action) => {
+export default (state = INITIAL_STATE, action: ContactsAction) => {
   switch (action.type) {
     case CONTACTS_UPDATE:
       return { ...state, contacts: action.payload };
```

Comment:
As far as I can tell this `CONTACTS_UPDATE` action has the exact same functionality as `CONTACTS_LOAD`, so maybe in a future PR all references to one should be converted to the other?